### DISCUSSION TITLE
macos版有可能推出arm原生版本嘛？

### DISCUSSION THREAD

**Discussion by juzijun233** [BeyondDimension/SteamTools#1132] [General]
如题

**Comment by AigioL** [comment]
Arm64 的版本编译出来运行闪退，目前还没办法解决

**Comment by AigioL** [comment]
目前提交触发的 Actions 中会编译 arm64 的存档包
这是当前最新的下载地址
[[Debug]Steam++_macos_arm64_86df7af](https://github.com/BeyondDimension/SteamTools/suites/5628544564/artifacts/183467597)
[[Release]Steam++_macos_arm64_86df7af](https://github.com/BeyondDimension/SteamTools/suites/5628544564/artifacts/183467601)

**Reply by juzijun233** [reply]
> 目前提交触发的 Actions 中会编译 arm64 的存档包 这是当前最新的下载地址 [[Debug]Steam++_macos_arm64_86df7af](https://github.com/BeyondDimension/SteamTools/suites/5628544564/artifacts/183467597) [[Release]Steam++_macos_arm64_86df7af](https://github.com/BeyondDimension/SteamTools/suites/5628544564/artifacts/183467601)

编译的包里面的文件好像不是.app 该咋运行呢

**Reply by AigioL** [reply]
使用这个脚本生成 .app
https://github.com/BeyondDimension/SteamTools/blob/develop/packaging/build-osx-app.sh
